### DISCUSSION TITLE
Create a submodule for resources?

### DISCUSSION THREAD

**Discussion by furtabs** [open-toontown/open-toontown#21] [Q&A]
Is this a possibility?

**Comment by LittleToonCat** [comment]
I've thought about that. While the concept of submodules is great, but that would add an extra step to the cloning process,  Besides, as far as I understand, most GUI-based Git tools won't automatically clone the submodules for you, you would have to do it yourself through command line `git clone --recurse-submodules git@github.com:open-toontown/open-toontown.git`

It would also add another step to pushing resource changes as well.  After pushing, you would also have to push a commit to the source repository stating to update the submodule.  And surprise surprise, You would have to update the submodule manually (outside the GUI) as well.

So I figured to save all that trouble and just straight up clone the resources into the source directory, but I'll leave this open to discussion though.
